lib/jekyll/admin/server/static_file.rb

```diff
@@ -27,6 +30,10 @@ class Server < Sinatra::Base
 
         private
 
+        def set_file_path
+          params["static_file_id"] = params["splat"].first
```

Comment:
Could this be done in a `before '/*' do` block? I'm not sure if the splats are processed that early in the request? If not, perhaps we should just update the `static_file_path` logic to look at the splat? I don't *think* `static_file_id` is used anywhere else (and everything in this namespace is private to this namespace).